front/src/App/OpenSpace/OpenSpace.js

```diff
@@ -104,6 +122,7 @@ const OpenSpace = () => {
       organizer,
       pendingQueue,
       slots,
+      activeCallForPapers,
```

Comment:
si esto es una variable booleana, yo seguiria la linea de Uncle Bob y la llamaria isActiveCallForPapers o hasActiveCallForPapers o shouldCallForPapers o isCallForPapersActive.

El dice que se use el prefijo is, should, has o similar como para que represente una respuesta si o no (booleana) y se distinga mas facil esas variables de otras (que tienen strings, numbers, etc).